src/oauth2.cc

```diff
@@ -139,6 +140,19 @@ std::string Sign(const std::string& data, const PKey& pkey) {
   return std::string(reinterpret_cast<char*>(result.get()), actual_result_size);
 }
 
+} // namespace
+
+OAuth2::OAuth2(const Environment& environment)
+  : OAuth2(environment, ExpirationImpl<std::chrono::system_clock>::New()) {}
+
+OAuth2::OAuth2(const Environment& environment,
+               std::unique_ptr<Expiration> expiration)
+  : environment_(environment),
+    token_expiration_(std::move(expiration)),
+    token_endpoint_(kDefaultTokenEndpoint) {
+  // Access the measure to ensure it is initialized. Otherwise, creating a view before
+  // any the first error would cause an error.
+  ::google::GceApiRequestErrors();
```

Comment:
Per other comments, IMO move this to RegisterViews.

Reply:
+1, this seems fragile. Can we not create the views on-demand right before they're needed?